src-ghc/Pact/ApiReq.hs

```diff
@@ -60,9 +60,19 @@ data ApiKeyPair = ApiKeyPair {
 instance ToJSON ApiKeyPair where toJSON = lensyToJSON 4
 instance FromJSON ApiKeyPair where parseJSON = lensyParseJSON 4
 
+
+data ApiPactId = ApiPactId {
```

Comment:
I don't think this is necessary, let's discuss.

Reply:
Doing away with this and just using Hash to represent the PactId might make it difficult for users to test continuation transactions. If users are using ApiReq to test their own things out, it'll be difficult for them to figure out what the PactId hash would be (unless they can easily hash things or can use our Haskell code). We could mitigate this by providing another tool like `pact -a` but for computing the pactId. Right now the `txHash` can be acquired by running `pact -a` on the transaction that initializes the pact.

Reply:
I kept "yield" in `continue-pact` but made it a fourth overloading so its a little better to use, aka
```
(continue-pact STEP)
(continue-pact STEP ROLLBACK)
(continue-pact STEP ROLLBACK PACT-ID)
(continue-pact STEP ROLLBACK PACT-ID YIELD)
```
Also added a simple test in `yield.repl` that makes use of it. @emilypi let me know if I should add more tests.

@slpopejoy, I mainly kept it in `continue-pact` instead of adding it to `(pact-state)` because `(pact-state)` does not default to the last pact initiated across all transactions, but instead it defaults to the last pact initiated in the current transaction. So if its a new transaction and a pact was initiated in a previous transaction, running `pact-state` will return a `PactExec not in context` error. I assumed we would like to test overwriting yield across any pacts initiated across different transactions, not just the ones in the current transaction.

Reply:
So, per our discussion, let's just go with the tx-hash of the first step.

Reply:
See latest commit.